src/_data/schemas/lpa.json

```diff
@@ -0,0 +1,289 @@
+                    }
+                },
+                "signature": {
+                    "$ref": "#/definitions/Signature"
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Certificate Provider"
+        },
+        "PersonToNotify": {
+            "type": "object",
+            "description": "The Person to Notify.",
+            "additionalProperties": false,
+            "properties": {
+                "actor": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Actor"
+                    }
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Person To Notify"
+        },
+        "ReplacementAttorney": {
+            "type": "object",
+            "description": "The Replacement Attorney.",
+            "additionalProperties": false,
+            "properties": {
+                "actor": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Actor"
+                    }
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Replacement Attorney"
+        },
+        "Actor": {
+            "type": "object",
+            "description": "Generic Actor that can be inherited across definitions.",
+            "additionalProperties": false,
+            "properties": {
+                "id": {
+                    "title": "Actor unique identifier",
+                    "description": "Identifies the Actor with a unique identifier.",
+                    "type": "integer",
+                    "format": "^[0-9]+$"
+                },
+                "title": {
+                    "title": "Title",
+                    "description": "The title given to an actor.",
+                    "type": "string"
+                },
+                "firstNames": {
+                    "title": "First Name",
+                    "description": "The actors first name.",
+                    "type": "string"
+                },
+                "lastName": {
+                    "title": "Last Name",
+                    "description": "The actors last name.",
+                    "type": "string"
+                },
+                "otherNames": {
+                    "title": "Other Names",
+                    "description": "Any other names for the actor.",
+                    "type": "string"
+                },
+                "dateOfBirth": {
+                    "type": "integer",
+                    "title": "Date of Birth",
+                    "maxLength": 13,
+                    "pattern": "^[0-9]{13}$",
+                    "description": "A 13 digit timestamp. Must be over the age of 18."
+                },
+                "address": {
```

Comment:
If we are storing addresses over time (i.e. as the LPA changes), do we need to capture that in the format somehow?

Reply:
I'd argue you want to be able to version the whole thing, not just the address. Names change too. I'm also not sure whether digitised LPAs have the same notion of locking as paper ones: would it be possible to edit the attorneys on an LPA, for example, up to the point where it's locked? You might want to keep that history, e.g. for the purposes of undo/redo.